troposphere/glue.py

```diff
@@ -481,6 +481,7 @@ class Action(AWSProperty):
     props = {
         'Arguments': (dict, False),
         'JobName': (basestring, False),
+        'CrawlerName': (basestring, False),
```

Comment:
Can you alphabetize and push an update? Thanks!

Reply:
Done :)

Reply:
I also came across some missing properties in the Condition, so I threw them in too.